scribusUnstable fails to install on nixos-unstable

## Issue description

`scribusUnstable` fails to install on `nixos-unstable`. It worked before the recent update. It fails late on in the build phase.

```
[ 89%] Generating ui_pdfimportoptions.h
[ 89%] Generating moc_importpdf.cpp
[ 89%] Generating moc_importpdfplugin.cpp
[ 89%] Generating moc_pdfimportoptions.cpp
Scanning dependencies of target importpdf
[ 89%] Building CXX object scribus/plugins/import/pdf/CMakeFiles/importpdf.dir/importpdf.cpp.o
In file included from /build/Scribus-r22730/scribus/plugins/import/pdf/importpdf.cpp:18:0:
/build/Scribus-r22730/scribus/plugins/import/pdf/slaoutput.h:31:10: fatal error: poppler/goo/gtypes.h: No such file or directory
 #include <poppler/goo/gtypes.h>
          ^~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [scribus/plugins/import/pdf/CMakeFiles/importpdf.dir/build.make:78: scribus/plugins/import/pdf/CMakeFiles/importpdf.dir/importpdf.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1763: scribus/plugins/import/pdf/CMakeFiles/importpdf.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 89%] Linking CXX shared module libimportpages.so
[ 89%] Linking CXX shared module libimportpct.so
[ 89%] Built target importpages
[ 89%] Built target importpct
[ 89%] Linking CXX shared module libimportodg.so
[ 89%] Built target importodg
make: *** [Makefile:152: all] Error 2
builder for '/nix/store/zjlpzcxymksw4l5p1kqdi9n9ycc3pz9r-scribus-unstable-2018-10-13.drv' failed with exit code 2
```

### Steps to reproduce

Try to install `scribusUnstable` on `nixos-unstable`.

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.13, NixOS, 19.03.git.626233e (Koi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2`
 - channels(root): `"nixos-19.03pre166308.626233eee6e"`
 - nixpkgs: `/pkgs/nixpkgs`

--- a/pkgs/applications/office/scribus/poppler-0.73.0.patch
+++ b/pkgs/applications/office/scribus/poppler-0.73.0.patch
@@ -1,0 +1,12 @@
+diff --git a/scribus/plugins/import/pdf/slaoutput.h b/scribus/plugins/import/pdf/slaoutput.h
+--- a/scribus/plugins/import/pdf/slaoutput.h
++++ b/scribus/plugins/import/pdf/slaoutput.h
+@@ -28,7 +28,7 @@ for which a new license (GPL+exception) is in place.
+ #include "selection.h"
+ #include "vgradient.h"
+ 
+-#include <poppler/goo/gtypes.h>
++#include <poppler/goo/gfile.h>
+ #include <poppler/Object.h>
+ #include <poppler/OutputDev.h>
+ #include <poppler/Gfx.h>

--- a/pkgs/applications/office/scribus/unstable.nix
+++ b/pkgs/applications/office/scribus/unstable.nix
@@ -16,6 +16,8 @@ stdenv.mkDerivation rec {
     sha256 = "18xqhxjm8dl4w3izg7202i8vicfggkcvi0p9ii28k43b5ps1akg1";
   };
 
+  patches = [ ./poppler-0.73.0.patch ];
+
   enableParallelBuilding = true;
 
   buildInputs = [

--- a/pkgs/applications/office/scribus/unstable.nix
+++ b/pkgs/applications/office/scribus/unstable.nix
@@ -4,16 +4,16 @@ podofo, poppler, poppler_data, python2, harfbuzz, qtimageformats, qttools }:
 
 let
   pythonEnv = python2.withPackages(ps: [ps.tkinter ps.pillow]);
-  revision = "22730";
+  revision = "22805";
 in 
 stdenv.mkDerivation rec {
   name = "scribus-unstable-${version}";
-  version = "2018-10-13";
+  version = "2019-01-14";
 
   src = fetchsvn {
     url = "svn://scribus.net/trunk/Scribus";
     rev = revision;
-    sha256 = "1nlg4qva0fach8fi07r1pakjjlijishpwzlgpnxyaz7r31yjaw63";
+    sha256 = "18xqhxjm8dl4w3izg7202i8vicfggkcvi0p9ii28k43b5ps1akg1";
   };
 
   enableParallelBuilding = true;

